server/openapi_server/test/test_File_mongo.py

```
@@ -0,0 +1,58 @@
+            file_content=b"Hello, world!",
+            owner_id=3222221111,
+            upload_date=datetime.now()
+        )
+        # Save the file to MongoDB
+        try:
+            result = save_file_to_mongo(cls.collection_name, cls.test_file)
+            if not result["acknowledged"]:
+                raise RuntimeError("MongoDB operation was not acknowledged.")
+            cls.inserted_id = result["inserted_id"]
+        except Exception as e:
+            raise RuntimeError(f"Failed to save file to MongoDB: {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)